codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/core/KotlinDependency.kt

```diff
@@ -118,6 +118,7 @@ data class KotlinDependency(
         val AWS_EVENT_STREAM = KotlinDependency(GradleConfiguration.Implementation, "$RUNTIME_ROOT_NS.awsprotocol.eventstream", RUNTIME_GROUP, "aws-event-stream", RUNTIME_VERSION)
         val AWS_PROTOCOL_CORE = KotlinDependency(GradleConfiguration.Implementation, "$RUNTIME_ROOT_NS.awsprotocol", RUNTIME_GROUP, "aws-protocol-core", RUNTIME_VERSION)
         val AWS_XML_PROTOCOLS = KotlinDependency(GradleConfiguration.Implementation, "$RUNTIME_ROOT_NS.awsprotocol.xml", RUNTIME_GROUP, "aws-xml-protocols", RUNTIME_VERSION)
+        val HTTP_AUTH_AWS = KotlinDependency(GradleConfiguration.Implementation, "$RUNTIME_ROOT_NS.http.auth", RUNTIME_GROUP, "http-auth-aws", RUNTIME_VERSION)
```

Comment:
question: Should the namespace be `$RUNTIME_ROOT_NS.http.auth.aws`?

Reply:
This was one of the questions I asked in the design doc. I agree that `aws` is already in the namespace and am generally against restating it. I could be convinced of `$RUNTIME_ROOT_NS.http.authaws` but at end of day re-using `http.auth` made most sense. We aren't too worried about collisions here since we own this namespace.